conf/deploy.json

```diff
@@ -1,19 +1,19 @@
 {
-    "defaultStacks": ["guardian-digital"],
+    "defaultStacks": ["guardian-digital-web"],
     "packages":{
-        "guardian-digital-website":{
+        "digital-department-website":{
             "type":"autoscaling",
             "data":{
-                "bucket": "guardian-digital-dist"
+                "bucket": "guardian-digital-web-dist"
```

Comment:
This doesn't line up with the UserData in your cloudformation script that downloads the artifact.